src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/from/index.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ESQLCommandOption } from '@kbn/esql-ast';
+import { isMarkerNode } from '../../../shared/context';
+import { metadataOption } from '../../../definitions/options';
+import type { SuggestionRawDefinition } from '../../types';
+import { getOverlapRange, handleFragment, removeQuoteForSuggestedSources } from '../../helper';
+import { CommandSuggestParams } from '../../../definitions/types';
+import {
+  isColumnItem,
+  isOptionItem,
+  isRestartingExpression,
+  isSingleItem,
+  sourceExists,
+} from '../../../shared/helpers';
+import {
+  TRIGGER_SUGGESTION_COMMAND,
+  buildFieldsDefinitions,
+  buildOptionDefinition,
+  buildSourcesDefinitions,
+} from '../../factories';
+import { ESQLSourceResult } from '../../../shared/types';
+import { commaCompleteItem, pipeCompleteItem } from '../../complete_items';
+import { METADATA_FIELDS } from '../../../shared/constants';
+
+export async function suggest({
+  innerText,
+  command,
+  getSources,
+  getRecommendedQueriesSuggestions,
+  getSourcesFromQuery,
+}: CommandSuggestParams<'from'>): Promise<SuggestionRawDefinition[]> {
+  // // TODO - this is a workaround because it was too difficult to handle this case in a generic way :(
+  // if (node && isSourceItem(node) && /\s/.test(node.name)) {
```

Comment:
Do we need to remove or uncomment this?

Reply:
Yeah, good catch I'm surprised a test isn't failing.